src/themes/greyVest/FilterListItem.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import { observer } from 'mobx-react'
+
+let FilterListItem = ({
```

Comment:
what is this used for?

Reply:
It turns out that this is the FilterItem for the GreyVest Picker components. It's distinct from the ListItem in general, as used in popovers and stuff, so I added a PickerItem theme key for it.